crowdsourcing/viewsets/task.py

```diff
@@ -123,22 +128,24 @@ def list_by_status(self, request, *args, **kwargs):
         for key, value in status_map.iteritems():
             task_workers = TaskWorker.objects.filter(worker=request.user.userprofile.worker, task_status=key)
             serializer = TaskWorkerSerializer(instance=task_workers, many=True,
```

Comment:
getting a permission error here with existing data on dashboard - "You do not have permission to perform this action."